composer.json

```diff
@@ -22,7 +22,7 @@
         }
     ],
     "require": {
-        "php": ">=8.1",
+        "php": ">=8.4",
```

Comment:
While I am using ~8.4.0 for all my packages, I know others more like `^8.4.0`. I would definitely not use `>= 8.4` because that would allow PHP 9 to also install the upcoming version while we don't know if we support PHP 9. But I don't know if I am alone on this one or not.

Reply:
Would you agree to go for `^8.4`? Then we don't have to do anything for PHP 8.5, but PHP 9 requires at least changing the composer.json.

Reply:
Yes I think thats resonable